New-PSPowerHour.ps1

```diff
@@ -0,0 +1,94 @@
+<#
+    .SYNOPSIS
+        Generate the markdown table of the presenters for the PSPowerHour
+    .NOTES
+        This utilizes PowerShellForGitHub module by the PowerShell team
+        For best results and the way written you will need a token from GitHub (no permissions needed to be assigned)
+#>
+[cmdletbinding()]
+param(
+    [string]$GitHubToken
+)
+
+function ConvertTo-Markdown {
```

Comment:
Is there any objection to using the [`PSMarkdown`](https://github.com/ishu3101/PSMarkdown) module here instead?